sdk/core/performance-stress/inc/azure/performance-stress/argagg.hpp

```diff
@@ -0,0 +1,1541 @@
+/*
```

Comment:
This is a lot of code for argument parsing.  Why not use a simple map<string, string>?

If you stay with this parser the liscense needs to be approved.

Reply:
This is for the application arguments from console line. The ones we get from the main(int, char **).  We can´t have a map there.

The license is MIT.

Reply:
@vhvb1989 This package needs to be approved before it can be used.

Reply:
MIT: https://github.com/vietjtnguyen/argagg

![image](https://user-images.githubusercontent.com/24213737/107081109-0c63d280-67a7-11eb-844d-b7834a85fbcf.png)